Engine/UserInput.cs

```diff
@@ -132,6 +133,13 @@ public class SubmitAttribute : Attribute { }
     /// <summary> Standard implementation of UserInputInterface for Command Line interfaces</summary>
     public class CliUserInputInterface : IUserInputInterface
     {
+        class Strings : IStringLocalizer
+        {
+            public string PleaseEnterNumberOrName => this.Translate("Please enter a number or name ");
```

Comment:
Should this pattern be like:

```cs
class Strings : IStringLocalizer
        {
           static Strings strings = new Strings();
           public static string PleaseEnterNumberOrName => strings .Translate("Please enter a number or name ");
```

Then it can be used as this:
```Strings.PleaseEnterNumberOrName```

Reply:
That's a great idea!